SPECS/babel/babel.spec

```diff
@@ -1,120 +1,100 @@
-%{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-
+Summary:        An integrated collection of utilities that assist in internationalizing and localizing Python applications
 Name:           babel
 Version:        2.6.0
-Release:        8%{?dist}
-Summary:        an integrated collection of utilities that assist in internationalizing and localizing Python applications
+Release:        9%{?dist}
 License:        BSD
-Group:          Development/Languages/Python
-Url:            https://babel.pocoo.org
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://babel.pocoo.org
 Source0:        https://files.pythonhosted.org/packages/be/cc/9c981b249a455fa0c76338966325fc70b7265521bad641bf2932f77712f4/Babel-%{version}.tar.gz
+BuildArch:      noarch
+
+%description
+Babel is an integrated collection of utilities that assist in internationalizing and localizing Python applications,
+with an emphasis on web-based applications.
 
-BuildRequires:  python2
-BuildRequires:  python2-devel
-BuildRequires:  python2-libs
-BuildRequires:  python-setuptools
-BuildRequires:  pytz
-BuildRequires:  pytest
-BuildRequires:  python3
+%package -n     python3-babel
+Summary:        An integrated collection of utilities that assist in internationalizing and localizing Python applications
 BuildRequires:  python3-devel
-BuildRequires:  python3-pytz
 BuildRequires:  python3-pytest
+BuildRequires:  python3-pytz
 BuildRequires:  python3-setuptools
 BuildRequires:  python3-xml
 %if %{with_check}
 BuildRequires:  curl-devel
 BuildRequires:  openssl-devel
-BuildRequires:  python-six
-BuildRequires:  python3-six
-BuildRequires:  python-attrs
 BuildRequires:  python3-attrs
+BuildRequires:  python3-six
 %endif
-Requires:       python2
-Requires:       python2-libs
-Requires:       pytz
-BuildArch:      noarch
+Requires:       python3
+Requires:       python3-pytz
 
-%description
+%description -n python3-babel
```

Comment:
OK, last time same comment about dropping the additional package and just making the default one be a Python 3 one. Let's just assume I copy-pasted it everywhere else, where it's applicable.:)

Reply:
Similar response to the related comment above.